automatic/zotero/update.ps1

```diff
@@ -11,32 +10,25 @@ function global:au_SearchReplace {
 
   @{
     ".\legal\VERIFICATION.txt" = @{
-      "(?i)(\s*1\..+)\<.*\>" = "`${1}<$($Latest.URL32)>"
-      "(?i)(^\s*checksum\s*type\:).*" = "`${1} $($Latest.ChecksumType32)"
-      "(?i)(^\s*checksum(32)?\:).*" = "`${1} $($Latest.Checksum32)"
-    }
-    ".\tools\chocolateyInstall.ps1" = @{
-      "(?i)^(\s*softwareName\s*=\s*)'.*'" = "`${1}'$softwareName'"
-      "(?i)(^\s*file\s*=\s*`"[$]toolsPath\\).*" = "`${1}$($Latest.FileName32)`""
-    }
-    ".\tools\chocolateyUninstall.ps1" = @{
-      "(?i)^(\s*softwareName\s*=\s*)'.*'" = "`${1}'$softwareName'"
-    }
-    ".\$($Latest.PackageName).nuspec" = @{
-      "(\<releaseNotes\>).*(\<\/releaseNotes\>)" = "`${1}https://www.zotero.org/support/$($version.Major).$($version.Minor)_changelog`${2}"
+      "(?i)(^\s*32-bit download\:).*" = "`${1} $($Latest.URL32)"
+      "(?i)(^\s*64-bit download\:).*" = "`${1} $($Latest.URL64)"
+      "(?i)(^\s*32-bit checksum\:).*" = "`${1} $($Latest.Checksum32)"
+      "(?i)(^\s*64-bit checksum\:).*" = "`${1} $($Latest.Checksum64)"
```

Comment:
You have removed the hard-coded values and the replacement of the software name.

Why has this been removed?

Reply:
I didn't realize having it hard-coded was important.  I would have thought just the opposite.  The more generic and self-referential, the more standard it is and useable across different packages.  

It seems redundant and inefficient to hard-code (and update?) the name while we have a `$env:ChocolateyPackageName` variable that matches.